lib/segmentation.js

```diff
@@ -1,3 +1,13 @@
+'use strict';
+
+/*
+ * Copyright (c) 2013-2021 node-coap contributors.
```

Comment:
I was wondering if the newly added files should also have a copyright that begins in 2013? On the other hand it probably doesn't make much of a difference.

Reply:
Formally a file copyright should start when file was added, but yes in the end ... no real dofference :-)